x-pack/plugins/reporting/server/lib/store/mapping.ts

```diff
@@ -45,7 +45,7 @@ export const mapping = {
   priority: { type: 'byte' },
   timeout: { type: 'long' },
   process_expiration: { type: 'date' },
-  created_by: { type: 'keyword' },
+  created_by: { type: 'keyword' }, // `null` if security is disabled
```

Comment:
Good callout

Reply:
Yeah, I think it used to be `false` but it sneakily changed through TS improvements. That shouldn't be a problem for mappings, I think.